tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -499,6 +499,8 @@ def __init__(self, file: str, config: Optional["WorkspaceConfig"]):
         self.context: chapel.Context = chapel.Context()
         self.file_infos: List["FileInfo"] = []
         self.global_uses: Dict[str, List[References]] = defaultdict(list)
+        self.instantiation_ids: Dict[chapel.TypedSignature, str] = {}
```

Comment:
I am surprised that TypedSignature can be used as the key for a dict, I guess it provides a hash method?

Reply:
I did specifically add one in a previous PR, it just hashes the PoiScope and TypedFnSignature pointers.